Getting-Started/Code/Source-Control/index.md

```diff
@@ -0,0 +1,139 @@
+
+** Nuget + Buildserver - if the plugin is installed via NuGet then as long as the packages.config file in the root of your project is source controlled, then the installed files for individual plugins shouldn't need to be source controlled (and your deployment process should pull the packages implementation files from Nuget during the deployment process).
+** Nuget + Manual deploy - here you don't necessarily need to source control the files, but you do need to remember to deploy them.
+** Backoffice - if your working with other developers, then it may be easier to add the plugin files to source control, or at least communicate to them that the particular package needs be installed via the backoffice
+** Backoffice + Buildserver - you'll need to include the plugin files in source control as the build server won't know to restore them - if the plugin/package doesn't come with a NuGet resource, consider setting up your own local NuGet repository for your build server to pull the files from.
+
+**Note**
+Each plugin could be different depending on what it's functionality is, and so may contain files it would be useful to track via Source control, and ones that should be ignored, check with the plugin's supporting website/developer for more information.
+
+### What folders and files should I **include** in my source control repository?
+
+#### Front-end build
+
+A lot depends on how you are running your front end build, eg you may be generating your css / js from SCSS or CoffeeScript, and it is the source of your frontend implementation that would need to be version controlled, allowing the actual css/js files to be generated during the build process, but generally speaking it's your Javascript, Css and Page Furniture eg /img and fonts - anything your site implementation uses statically that you would add to version control.
+
+#### Views/Templates/Partials
+
+Umbraco site templates/views can be edited via the Umbraco Backoffice, but ultimately they reside in the **/views** folder on disk (or **/masterpages** on webforms implementations), because these views/template often include code, it can make an awful lot of sense to have their changes tracked under source/version control.
+However this can pose a problem in the common scenario where the line is blurred between an 'editor' and an 'implementor/developer' - where you might have individuals comfortable with making template changes via the backoffice, but not so comfortable with using a source control repository such as GIT.
+Umbraco Cloud is a great solution in these scenarios, as changes via the backoffice are tracked in a Git repository automatically - otherwise filewatchers and events, to alert when changes are made via the backoffice, or more simply, taking a copy of the production version of views into Source Control at the beginning of any development sprint.
```

Comment:
What a sentence. To be honest, this page is about NON CLOUD.  I would leave it.
As you will have other issues too when having a VS solution with multiple projects.